check-lead-picture.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -eu
+
+lang="en es ja cz fr pt pl de it ko zh tr cn"
+
+if ! cmd=$(command -v curl)
+then
+  echo "please install curl"
+  exit 1
+fi
+
+if (( $# == 0 ))
+then
+  feed="https://weeklyosm.eu/feed"
+  article_id=$(curl -Ls "$feed" | grep "<link>https://weeklyosm.eu/archives/" | head -n 1 | cut -d "<" -f 2 | grep -o '[^/]*$')
+else
+  article_id=$1
```

Comment:
we are calling our script with [parameters](https://github.com/call-me-matt/weeklyOSMnotifier/blob/a09b9d20e578e0dcff29bc8c91441c6519cb78d5/README.md?plain=1#L79) - so I guess this would always go this path. $1 would be the first parameter, which is not the article id at the moment.